### PR TITLE
Add admin to code owners of LICENSE, CODE OF CONDUCT etc

### PR DESCRIPTION
**Pull request by banesullivan**
Some files that if modified, need to be reviewed by the @pyvista/admin  team

### CODE REVIEW

**Review by tkoyama010 — APPROVED** [review]
LGTM

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/LICENSE                                            @pyvista/admin
+/AUTHORS.rst                                        @pyvista/admin
+/joss/                                              @pyvista/admin
```
Comment (tkoyama010) [inline .github/CODEOWNERS:3]:
```suggestion
/joss/                                              @pyvista/admin
/CODE_OF_CONDUCT.md                                 @pyvista/admin
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

**Comment by banesullivan** [comment]
Also, I want to try 😎: @pyvista-bot LGTM

**Comment by banesullivan** [comment]
@pyvista-bot LGTM

**Comment by banesullivan** [comment]
Shucks, @pyvista-bot LGTM isn't working

**Comment by banesullivan** [comment]
Oh wait no... it may be working. I just checked the workflow logs and it is very backlogged

**Comment by tkoyama010** [comment]
> Oh wait no... it may be working. I just checked the workflow logs and it is very backlogged

@banesullivan We maybe need #5314 . @germa89 taught me that we should remove comma(,) in yaml file in https://github.com/ansys/pymapdl/pull/2229#discussion_r1421674714 .

**Comment by tkoyama010** [comment]
> ✅ Approving this PR because [banesullivan](https://github.com/banesullivan) said so in [here](https://github.com/pyvista/pyvista/pull/5327#issuecomment-1851365491) 😬

Oops, looks like it worked correctly even with the comma. Perhaps there were too many other Actions, causing a wait. @germa89 It seems to work fine even with the comma. Please refer to this in your project as well.

**Review by tkoyama010 — APPROVED** [review]

**Review by tkoyama010 — APPROVED** [review]

**Pull request merged**